experiments/modelfree/noisy_actions_and_obs.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+function wait_proc {
+    if [[ -f ~/ray_bootstrap_config.yaml ]]; then
+        # Running on a Ray cluster. We want to submit all the jobs in parallel.
+        sleep 5  # stagger jobs a bit
+    else
+        # Running locally. Each job will start a Ray cluster. Submit sequentially.
+        wait
+    fi
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+OUT_ROOT=data/aws/score_agents
+TIMESTAMP=`date --iso-8601=seconds`
+
+MULTI_SCORE_CMD="python -m modelfree.multi.score with medium_accuracy "
+
+# Not sure what this is doing
```

Comment:
Kill lines 19-23. You don't use log directories in this script. (FYI, the if statement is checking if there are no CLI arguments, in which case it exits abnormally printing a usage message.)

Reply:
Sounds good. This was very much a Chesterton's Fence part of the script for me.